src/index.js

```diff
@@ -281,7 +281,7 @@ function friendActivityData(date) {
   let friends = [];
   let findFriends = userRepository.getFriends();
   findFriends.forEach(friend => {
-    let friendData = activityRepository.getUserLogs(friend);
+    let friendData = activityRepository.getUserLogs(friend, "activityData");
     let friendName = userRepository.getUserData(friend).name;
     let indexDay = friendData.findIndex(user => user.date === date);
     let friendWeeks = friendData.slice(indexDay - 6, indexDay + 1);
```

Comment:
I checked this out locally on my machine, and can confirm that all tests are passing, and all data is displayed on the dom.